bin/express

```diff
@@ -65,7 +66,7 @@ var www = fs.readFileSync(__dirname + '/../templates/js/www', 'utf-8');
     if (empty || program.force) {
       createApplicationAt(path);
     } else {
-      program.confirm('destination is not empty, continue? ', function(ok){
+      prompt.confirm('destination is not empty, continue? ')(function(err,ok){
```

Comment:
I just read the source code for `co-prompt` and, in the latest version, it's not possible for `err` to contain anything other than `null`, but I'm sure it was put there as a matter of a standard interface. I still think it should be handled anyway, though :) A `if (err) throw err` should be good enough for this command line program.